src/main/scala/is/hail/utils/FunctionDocumentation.scala

```diff
@@ -89,11 +95,14 @@ case class DocumentationEntry(name: String, category: String, objType: Option[Ty
   val objCategory = {
     objType match {
       case Some(ot) => ot match {
-        case x: TAggregable => Some("Aggregable")
-        case x: TAggregableVariable => Some("Aggregable")
-        case x: TArray => Some("Array")
-        case x: TSet => Some("Set")
-        case x: TDict => Some("Dict")
+        case TAggregable(_) => Some("Aggregable")
+        case TAggregableVariable(_, _) => Some("Aggregable")
+        case TArray(_) => Some("Array")
+        case TSet(_) => Some("Set")
+        case TDict(_, _) => Some("Dict")
+        case TVariant(_) => Some("Variant")
+        case TLocus(_) => Some("Locus")
```

Comment:
These lines are causing the section header (shows up in TOC on left sidebar) to be "Variant" instead of "Variant(GR)". If you prefer "Variant(GR)", let me know. I thought for now, this would be better. If we switch it to "Variant(GR)", then I'll need to fix all of the Sphinx references to Variant, etc.

Reply:
OK, so I looked over the docs, and the only confusing thing is that GR is used type signatures for the methods on the classes, but it isn't defined anywhere, so that's confusing.  So I think we either need to define it in the next or put it in the type name.  Putting it in the type name matches better things like `Array[T]` which seems good.